assets/pages/monthly-contributions/monthlyContributions.jsx

```diff
@@ -37,22 +40,23 @@ pageStartup.start();
 
 // ----- Redux Store ----- //
 
-const store = createStore(reducer, {
-  intCmp: getQueryParameter('INTCMP'),
-}, applyMiddleware(thunkMiddleware));
-
-
-// Retrieves the contrib amount from the url and sends it to the redux store.
-const contributionAmount = getQueryParameter('contributionValue', '5');
+const contributionAmount = Number(getQueryParameter('contributionValue')) || 5;
+const country = Country.detect();
+const currency = Currency.forCountry(country);
 
-if (contributionAmount !== undefined && contributionAmount !== null) {
-  store.dispatch(setContribAmount(contributionAmount));
-}
+/* eslint-disable no-underscore-dangle */
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
This is the browser extension version, so I think it should have no impact on the bundle size or performance (unless you have the plugin installed)